coralproject/talk
# User integration

## What does this PR do?

Integrates user information into comments and actions.

## How do I test this PR?

1. Open the comment stream.
2. You should not be able to like or flag.
3. Log in.
4. You should be able to post a comment. This comment should have the right name.
5. Like and flag a comment.
6. Check admin panel, the post requests to the server should include your user id.
7. Click the "Reply" button on a reply. The reply box should open on the parent comment.
8. Create a stream with several comments and replies. Like buttons should still work.
9. Click the "flag" button and reload the page. The like count on that comment should be unchanged.
10. Post a comment. The notification should not cover the last comment in the stream.

@coralproject/tech
